chapter_convolutional-modern/alexnet_vn.md

```diff
@@ -92,7 +100,14 @@ In fact, engineering a new set of feature functions, improving results, and writ
 and similar feature extractors ruled the roost.
 -->
 
-*dịch đoạn phía trên*
+Cách khác để kết nối trạng thái của các dữ kiện liên quan với nhau, đó cũng là phần quan trọng nhất của pipeline là thực hiện biểu diễn.
```

Comment:
```suggestion
Một cách nhìn khác về tình trạng tại thời điểm đó là các biểu diễn là phần quan trọng nhất của pipeline.
```

Reply:
Chỗ này sửa lại đọc thấy tối nghĩa quá mà không hiểu gì luôn @lkphuc ạ

Reply:
:)) đúng vậy thật, cơ mà như cách dịch gốc thì ko đúng nghĩa lắm 
```suggestion
Nói một cách khác, tại thời điểm đó phần lớn các nhà nghiên cứu tin rằng phần quan trọng nhất của pipeline là các biểu diễn.
```

Reply:
Trích lại nguyên câu.
"Another way to cast the state of affairs is that the most important part of the pipeline was the representation."  
Câu này mình hiểu nếu chia thành hai câu như sau:
Cách khác để kết nối trạng thái các dữ kiện lúc đó là việc biểu diễn. 
Việc kết nối này là phần quan trọng nhất của pipeline.

Reply:
Another way to cast the state of the affairs tức là cách nói khác, cách nhìn khác của câu này ở ngay phía trên nó ý:
"Những nhà nghiên cứu thị giác máy tính thời đó có lý do để tin rằng một tập dữ liệu hơi lớn hơn hay sạch hơn hoặc một pipeline trích chọn đặc trưng tốt hơn một chút sẽ có ảnh hưởng lớn hơn bất kỳ giải thuật huấn luyện nào."

Tức là họ đang paraphrase lại câu trên thôi. 
Còn em không hiểu 2 ý này lắm: `Cách khác để kết nối trạng thái các dữ kiện lúc đó là việc biểu diễn. Việc kết nối này là phần quan trọng nhất của pipeline.`

Reply:
em thấy đoạn phía trước mô tả cách làm truyền thống, xử lý đặc trưng thủ công, xem trọng algorithms. Câu cuối gợi mở hướng nhìn khác:
> Những nhà nghiên cứu thị giác máy tính thời đó có lý do để tin rằng một tập dữ liệu hơi lớn hơn hay sạch hơn hoặc một pipeline trích chọn đặc trưng tốt hơn một chút sẽ có ảnh hưởng lớn hơn bất kỳ giải thuật huấn luyện nào

Và câu đầu đoạn này `Another way to ...` sẽ support cho việc xem trọng các đặc trưng và các biểu diễn của nó, chứ không phải xem trọng algorithms như traditional ML researchers ở đoạn trước. Em nghĩ ý cả câu là:
```suggestion
Một cách nhìn khác là coi phần quan trọng nhất của pipeline là thực hiện biểu diễn.
```
Bỏ đi hơi nhiều, mong mọi người góp ý.

Reply:
Thực hiện biểu diễn có thể hình dung giống như khi phân loại đối tượng chẳng hạn như trong thuật toán SVM, khi giữa các đối tượng data không tuyến tính ở 2 chiều, ta có thể nâng cấp data thành 3 chiều và từ đó các lớp đối tượng lại có thể phân tách tuyến tính. Nếu mình hiểu đúng thì tác giả đang ám chỉ việc này.

Reply:
@LongNguyen1984 cái anh đang nói là kernel trick trong SVM, cũng thuộc cách xử lý đặc trưng thủ công truyền thống. `Thực hiện biểu diễn` em hiểu là tìm cách biểu diễn các đặc trưng. Cách làm ML truyền thống không xem trọng việc này mà quan tâm nhiều về thuật toán. Ở đây tác giả đang muốn chuyển tiếp sang hướng đi khác là trực tiếp học các đặc trưng này (học cách biểu diễn). Câu này mở đầu đoạn chuyển tiếp nên logic mà nói nó nên nhấn mạnh tầm quan trọng của biểu diễn.

Reply:
Em ko follow 2 comments của mọi người lắm.
Theo em hiểu em chỉ đang ko đồng tình cách dịch chữ `Another way to cast the state of affair...` và em khá tự tin rằng nó chỉ ý: Một cách nói mô tả khác về tình hình cộng đồng nghiên cứu chung thời đó coi pipeline feature engineer là quan trọng nhất. 
https://www.collinsdictionary.com/dictionary/english/state-of-affairs 
Vậy nên em vẫn giữ suggestion của em 2 ngày trước.

Reply:
> @LongNguyen1984 cái anh đang nói là kernel trick trong SVM, cũng thuộc cách xử lý đặc trưng thủ công truyền thống. `Thực hiện biểu diễn` em hiểu là tìm cách biểu diễn các đặc trưng. Cách làm ML truyền thống không xem trọng việc này mà quan tâm nhiều về thuật toán. Ở đây tác giả đang muốn chuyển tiếp sang hướng đi khác là trực tiếp học các đặc trưng này (học cách biểu diễn). Câu này mở đầu đoạn chuyển tiếp nên logic mà nói nó nên nhấn mạnh tầm quan trọng của biểu diễn.

Câu "Cách làm ML truyền thống không xem trọng việc này mà quan tâm nhiều về thuật toán" là hoàn toàn ngược lại em nhé. ML truyền thống luôn dùng hand-craft features, lựa chọn feature là một trong những yêu cầu tiên quyết khi làm ML.